### DISCUSSION TITLE
Welcome to antidote Discussions!

### DISCUSSION THREAD

**Discussion by Finistere** [Finistere/antidote#40] [Announcements]
Welcome!

**Comment by pauleveritt** [comment]
Thanks for setting this up! @flisboac You ok with moving some of the chatting here?

**Reply by flisboac** [reply]
Of course! Here is a better place.

@Finistere should I migrate the catalog idea as a new discussion?

**Reply by Finistere** [reply]
Would be great if you could! Just point to the issue, eventually copy-pasting your last message(s). That would be more than enough :)
